src/main/java/ca/on/oicr/gsi/visionmate/mockServer/MockScannerServer.java

```diff
@@ -17,67 +17,65 @@
 /**
  * Simple socket server to mock the behaviour of the VisionMate TCP/IP server for testing purposes. Intended to run in a thread:
  * <p>
+ * 
  * <pre>
- * {@code
- * MockScannerServer server = new MockScannerServer();
- * new Thread(server).start();
+ * {
+ *   &#64;code
```

Comment:
Please fix. Might want to either make your default auto-formatter less aggressive, or just not use a default auto-formatter